src/components/policy/policy_regular/src/sql_pt_representation.cc

```diff
@@ -1414,10 +1414,9 @@ bool SQLPTRepresentation::SaveModuleMeta(const policy_table::ModuleMeta& meta) {
   }
   const int64_t odometer = *(meta.pt_exchanged_at_odometer_x);
 
-  query.Bind(0, *(meta.ccpu_version));
```

Comment:
@ydementieiev SaveModuleMeta  () does not need ccpu_version? Why?

Reply:
@AByzhynar we saved ccpu_version on SetMetaInfo() function, this line was added by me [here](https://github.com/LuxoftSDL/sdl_core/commit/309671f16072568421cf22f57d7747b78331f938), but they are not needed